README.md

```diff
@@ -132,3 +149,31 @@ You can also specify the path into which you want to clone the repository:
     renode-run: python --version
     repos: https://github.com/antmicro/pyrav4l2.git folder1
 ```
+
+## Network
+
+You can disable networking in the emulated Linux by passing the `netowrk: false` argument to the action
+
+```yaml
+- uses: antmicro/renode-linux-runner-action@v0
+  with:
+    shared-dir: ./shared-dir
+    renode-run: python --version
+    network: false
+```
+
+This can be useful when running the action in the Matrix Docker container strategy if you do not have permission to create `tap` interfaces.
```

Comment:
```suggestion
This can be useful when running the action in a container matrix strategy if you do not have permission to create `tap` interfaces.
```